docs/index.rst

```diff
@@ -19,7 +19,7 @@ Education should be collaborative
 
 Students know best
 ~~~~~~~~~~~~~~~~~~
-    Universities can't keep up with technology. Most university systems aren't even mobile responsive! Forget about using social media. That's why Semester.ly is built by students, and always will be. That's why we're open source! Oh, and it's why we use emojis 🙂.
```

Comment:
Why remove this line?

Reply:
Read the commits~~

I said it's because it feels out of place in an otherwise formal pitch

Reply:
If you can note to read commit msgs if they're particularly useful, please do so in the future.

We can leave it out I suppose, but we should maintain some level of informality on the site itself just for future reference.

Reply:
> we should maintain some level of informality on the site itself just for future reference.

What do you mean by this?

Reply:
Typically, it is not expected to read through commit messages and reading code. 
It's more concise if it could be written in brief in the description or the PR description makes a reference to thoroughly read the commit messages.